cli/src/commands/mod.rs

```diff
@@ -3019,7 +3019,7 @@ fn cmd_resolve(
         );
     };
 
-    let (repo_path, _) = conflicts.get(0).unwrap();
+    let (repo_path, _) = conflicts.first().unwrap();
```

Comment:
I think this will conflict with #2449, and since that PR seems more annoying to regenerate, I say we let it get merged first.

Reply:
I'm not sure if there will be a follow-up to #2449 moving `cmd_resolve` to a separate file, but it's not here yet. Either way, I'm happy to wait longer or not, whatever you prefer.

Reply:
I'd think so, but let's hear what @AntoineCezar says

Reply:
I rebased this on top of the move of `resolve` to a separate file.